scripts/travis.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+npm run install:packages
+
+export CLIENT_ID=0oad5a19u0IVnKU1g0h7
+export REDIRECT_URI=http://localhost:8080/implicit/callback
+
+cd packages/jwt-verifier
+npm test > ./jwt-verifier-test.log
+
+if (cat ./jwt-verifier-test.log | grep -i -e "failed:" -e "failing" > /dev/null); then
```

Comment:
Why are we manually doing this error handling logic?  Shouldn't the `npm test` command bubble it's error upwards?  Also, why are we manually invoking tests rather than using `lerna run test` and pushing the test configuration down to each package?

Reply:
+1.

We could check for any non-zero exit code by using `||` :
`npm test || echo "Tests Failed!"`